recipes/_efa_install.rb

```diff
@@ -0,0 +1,39 @@
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: _efa_install
+#
+# Copyright 2013-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Install only on Centos 7
+if node['platform'] == 'centos' && node['platform_version'].to_i >= 7
+  efa_installer = "#{node['cfncluster']['sources_dir']}/aws-efa-installer-latest.tar.gz"
+
+  # Get EFA Installer
+  remote_file efa_installer do
+    source node['cfncluster']['efa']['installer_url']
+    mode '0644'
+    retries 3
+    retry_delay 5
+    not_if { ::File.exist?(efa_installer) }
+  end
+
+  bash "install efa" do
+    cwd node['cfncluster']['sources_dir']
+    code <<-NODE
+      yum remove -y openmpi
```

Comment:
Could you add a comment describing why we need to remove openmpi?

Reply:
Added